### ISSUE TITLE
Error in Readme: npx git-cz installs wrong package

### ISSUE THREAD

**Issue by EricLambrecht** [commitizen/cz-cli#564]
In the Readme it says

> Alternatively, if you are using NPM 5.2+ you can use npx instead of installing globally:
> `npx git-cz`

This will install [this](https://github.com/streamich/git-cz) package though (streamich/git-cz).
Is this intended? The package doesn't seem related to the official *commitizen* team.

**Note:** Commitizen will still work, but it will use the *streamich/git-cz* emoji adapter.

**Comment by jimthedev** [comment]
Yeah, you are correct. Looks like we will need to figure out how to work with npx since our repo, package and binary names are all different 🤕

**Comment by andrico1234** [comment]
@EricLambrecht @jimthedev 

I've found a workaround to this issue. 

I don't like installing globalsm so in my package.json file I've got everything set up like this.
```
{
  "name": "demo",
  "version": "0.1.0",
  "scripts": {
    "commit": "yarn git-cz",
  },
  "devDependencies": {
    "commitizen": "^3.0.2",
    "conventional-changelog-cli": "^2.0.5",
    "cz-conventional-changelog": "^2.1.0",
  },
  "config": {
    "commitizen": {
      "path": "cz-conventional-changelog"
    }
  },
}
```

Running `yarn commit` works with the correct version of git-cz. This is because of the script we're running in here: `commit: yarn git-cz`. We're not using npx so there's no possibility of pulling in the incorrect git-cz module. 

My suggestion would be to let people know in the README.md that there is confusion between two versions of git-cz. This left me stumped for the most part of the day, and knowing that commitizen has it's own internal git-cz script separate streamich's one, would've saved me a shit-ton of headscratching
